opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -389,7 +407,12 @@ def add_event(
     ) -> None:
         pass
 
-    def add_lazy_event(self, event: Event) -> None:
+    def add_lazy_event(
+        self,
+        name: str,
+        event_formatter: types.AttributesFormatter,
+        timestamp: typing.Optional[int] = None,
+    ) -> None:
         pass
 
```

Comment:
Do we need a method for `add_lazy_link` and `add_link`?

Reply:
Links can only be added when creating the span: https://github.com/open-telemetry/opentelemetry-python/pull/259.